frontend/packages/kubevirt-plugin/src/components/vms/vm-console.tsx

```diff
@@ -79,6 +87,25 @@ const VMConsoles: React.FC<VMConsolesProps> = ({
 
   const vncServiceManual = (vnc && vnc.manual) || undefined;
   const rdpServiceManual = (rdp && rdp.manual) || undefined;
+  const vmName = getName(vm);
+  const namespace = getNamespace(vm);
+
+  const onExternalWindowClick = () => {
+    newWin.current = window.open(
+      `/vm-console?namespace=${namespace}&name=${vmName}`,
+      '',
+      `width=1024,height=768,left=200,top=200`,
+    );
+
```

Comment:
IMHO we shouldn't create an additional logic for this. We should have the same behaviour when somebody opens the console in the new tab by themselves.

Can we just create a simple link with `href="/vm-console" target="_blank" rel="noopener noreferrer"` ?

Current console will get `Disconnected` by default in the current window

Reply:
From my experience they interfere each other.
also having a variable for the `window` helps us know if it got blocked by an Adblocker or not.

Reply:
What do you mean by interfering? You can still open a new tab with `/vm-console`.

You don't have to care about adblock when you have link for a new tab